client/app/components/user/session/user-session.service.js

```diff
@@ -531,13 +531,6 @@ class SessionService {
             click: () => this.trackUserMenuSection('my_contacts', 'contacts'),
           },
 
-          // Tickets
-          {
-            title: this.$translate.instant('menu_tickets'),
```

Comment:
`menu_tickets ` can be removed ?

Reply:
@FredericEspiau Actually that translation key is used in multiple place so I cannot remove it safely. That's why I have keeped it.